apis/datadoghq/v1alpha1/datadogagentprofile_types.go

```diff
@@ -48,6 +49,10 @@ type Override struct {
 	// +optional
 	PriorityClassName *string `json:"priorityClassName,omitempty"`
 
+	// The deployment strategy to use to replace existing pods with new ones.
+	// +optional
+	UpdateStrategy *v2alpha1.UpdateStrategy `json:"updateStrategy,omitempty"`
```

Comment:
Should we use v2alpha1 directly or move `UpdateStrategy` to [common](https://github.com/DataDog/datadog-operator/blob/main/apis/datadoghq/common/v1/types.go)?

Reply:
Agree, thanks for calling out would have overlooked. I'll create a PR.